compiler/p/codegen/OMRMemoryReference.cpp

```diff
@@ -1068,375 +1068,329 @@ void OMR::Power::MemoryReference::mapOpCode(TR::Instruction *currentInstruction)
       }
    }
 
-
-uint8_t *OMR::Power::MemoryReference::generateBinaryEncoding(TR::Instruction *currentInstruction, uint8_t *cursor, TR::CodeGenerator *cg)
+TR::Instruction *OMR::Power::MemoryReference::expandForUnresolvedSnippet(TR::Instruction *currentInstruction, TR::CodeGenerator *cg)
```

Comment:
Referencing [4] from eclipse/openj9#9113. Can you confirm that the binary encoding piece which used to handle the unresolved case has now moved here?

Reply:
Yes, this is where the code that previously handled the unresolved case during binary encoding has moved to.